test/Datadog.Trace.ClrProfiler.IntegrationTests/CustomTestFramework.cs

```diff
@@ -3,188 +3,20 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
 
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-using System.Reflection;
-using System.Threading;
-using System.Threading.Tasks;
-using Datadog.Trace.TestHelpers;
+using Datadog.Trace.ClrProfiler;
 using Xunit;
 using Xunit.Abstractions;
 using Xunit.Sdk;
 
 [assembly: TestFramework("Datadog.Trace.ClrProfiler.IntegrationTests.CustomTestFramework", "Datadog.Trace.ClrProfiler.IntegrationTests")]
 
-namespace Datadog.Trace.ClrProfiler.IntegrationTests
+namespace Datadog.Trace.Security.IntegrationTests
```

Comment:
Are you sure the namespace should change?

Reply:
good point, I mixed them up!